### PR TITLE
use setUpClass in BaseAccountingTest

### PR DESCRIPTION
**Pull request by dannyroberts**

### CODE REVIEW

**Comment by czue** [comment]
I think at least one of these test failures is real:

```
======================================================================
FAIL: test_dry_run (corehq.apps.hqadmin.tests.test_prbac.TestCchqPrbacBootstrap)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/hqadmin/tests/test_prbac.py", line 23, in test_dry_run
    self.assertEquals(Role.objects.count(), 0)
AssertionError: 24 != 0
----------------------------------------------------------------------
```

File: corehq/apps/hqadmin/management/commands/cchq_prbac_bootstrap.py [hunk 1]
```diff
@@ -29,30 +29,39 @@ class Command(BaseCommand):
                     help='Enable debug output'),
         make_option('--fresh-start', action='store_true',  default=False,
                     help='We changed the core v0 plans, wipe all existing plans and start over. USE CAUTION.'),
+        make_option('--flush', action='store_true',  default=False,
```
Comment (dimagimon) [inline corehq/apps/hqadmin/management/commands/cchq_prbac_bootstrap.py]: E241 multiple spaces after ','

File: corehq/apps/hqadmin/management/commands/cchq_prbac_bootstrap.py [hunk 2]
```diff
@@ -29,30 +29,39 @@ class Command(BaseCommand):
...
+                    help='Wipe all roles and start over. USE CAUTION.'),
+
         make_option('--testing', action='store_true',  default=False,
                     help='Run this command for tests.'),
     )
 
-    def handle(self, dry_run=False, verbose=False, fresh_start=False, testing=False, *args, **options):
+    def handle(self, dry_run=False, verbose=False, fresh_start=False, testing=False, flush=False, *args, **options):
```
Comment (dimagimon) [inline corehq/apps/hqadmin/management/commands/cchq_prbac_bootstrap.py]: E501 line too long (116 > 115 characters)

File: corehq/apps/hqadmin/management/commands/cchq_prbac_bootstrap.py [hunk 3]
```diff
@@ -29,30 +29,40 @@ class Command(BaseCommand):
                     help='Enable debug output'),
         make_option('--fresh-start', action='store_true',  default=False,
                     help='We changed the core v0 plans, wipe all existing plans and start over. USE CAUTION.'),
-        make_option('--testing', action='store_true',  default=False,
+        make_option('--flush', action='store_true',  default=False,
```
Comment (dimagimon) [inline corehq/apps/hqadmin/management/commands/cchq_prbac_bootstrap.py]: E241 multiple spaces after ','

**Comment by dannyroberts** [comment]
screw it

**Comment by czue** [comment]
?

**Comment by czue** [comment]
(last commit was :white_check_mark: )

**Comment by dannyroberts** [comment]
Ha, I got confused by an unrelated local test failure right when I'd finally fixed things... This really doesn't seem to have sped the accounting tests up at all on travis which is disappointing. I'll let this one run again and look, but if we don't see an improvement then it's not worth it because this code is slightly more complicated

**Comment by czue** [comment]
@dannyroberts what's the status of this?

**Comment by dannyroberts** [comment]
Thanks, gonna just close it: I think I decided to abandon it due to lack of compellingness of the results.

**Pull request closed without merging**